pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -537,6 +540,10 @@ in
     nvimRequireCheck = "cmp_tmux";
   };
 
+  cmp-vimtex = super.cmp-vimtex.overrideAttrs {
+    dependencies = [ self.nvim-cmp ];
```

Comment:
I thought we wanted to avoid that, as it can be used by blink.cmp too ?

Reply:
I'm fine with either, I didn't see a concrete opinion on https://github.com/NixOS/nixpkgs/pull/358809#issuecomment-2496254619 so left it in here for now until that one was resolved. If we want to merge that and then just skip checking on these completions, I can do that instead.